apps/cookbook/src/app/showcase/list-virtual-scroll-showcase/list-virtual-scroll-showcase.component.ts

```diff
@@ -20,7 +20,7 @@ export class ListVirtualScrollShowcaseComponent {
       defaultValue: 'false',
     },
     {
-      name: 'viewPortHeight',
+      name: 'virtualScrollViewportHeight',
       description: 'A fixed viewport height in pixels.',
```

Comment:
This states what it is - but not really what it is used for or what effect it has setting this. Consider rewording it to be more like the other two descriptions 😄